independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/update/QuarkusUpdateRecipe.java

```diff
@@ -0,0 +1,76 @@
+package io.quarkus.devtools.project.update;
+
+
+import io.quarkus.devtools.project.BuildTool;
+import io.quarkus.devtools.project.update.operations.UpdatePropertyOperation;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class QuarkusUpdateRecipe {
+
+    public static final Map<String, Object> QUARKUS_RECIPE = Map.of(
```

Comment:
any reason to not just make a proper class with type,name, etc. as properties?

Reply:
just simpler as Map doesn't have to be configured in SnakeYaml